src/main/java/locators_constants/MoodUpdatedModalConstants.java

```diff
@@ -0,0 +1,5 @@
+package locators_constants;
+
+public interface MoodUpdatedModalConstants {
+    String BUTTON_MY_DIARY = "//*[@class='btn btn-sm btn-default ButtonMyDiary pull-left']";
```

Comment:
don't include all classes in xpath, we discussed that on previous lessons,  better to use something like that:
//*[contains(@class, 'ButtonMyDiary')]

Reply:
done